node/src/services/handler.rs

```diff
@@ -21,6 +25,7 @@ pub trait StorageHubHandlerConfig: StorageHubHandlerInitializer + Send + 'static
     ///
     /// This is primarily used for constructing
     type TrieLayout: TrieLayout;
+
```

Comment:
Should this be here? Seems inconsistent with the other spacing used in other fields.